scripts/install-worker.sh

```diff
@@ -73,8 +72,20 @@ sudo yum install -y \
   mdadm \
   pigz
 
-# Remove any old kernel versions. `--count=1` here means "only leave 1 kernel version installed"
-sudo package-cleanup --oldkernels --count=1 -y
+# packages that need special handling
+SPECIAL_PACKAGES=(
+  "curl"            # exists in both AL2 and AL2023 (need to overwrite curl-minimal)
+  "iptables-legacy" # exists in AL2023 only (needed by kubelet)
+)
+
+for package in "${SPECIAL_PACKAGES[@]}"; do
+  (yum list available | grep -qw $package) || sudo yum -y --allowerasing install $package
```

Comment:
s/`yum`/`dnf`/

although there is a symlink so it still works...

Reply:
ack